bin/inchroot.sh

```diff
@@ -92,6 +92,7 @@ adduser user --disabled-password --gecos user
 # TODO: Set the password for "user"
 mkdir -p /home/user/Desktop
 chown user:user /home/user/Desktop
+chmod 755 /home/user                 ## ticket #2371
```

Comment:
I am not sure if home user folder permissions are related to this issue.
Does QGIS Server read anything from the user home folder?

Reply:
@kalxas Well, now I am building `darkblue-b:qgis-serv7` image locally, so I can't launch another VM, but QGIS Server example reads National Earth 2 data from `/home/user/data/xxx` as symbolic link or something...
https://github.com/OSGeo/OSGeoLive/blob/master/app-data/qgis-server/themes.json#L80

Reply:
Well, the following QGIS project file reads shape files from `/home/user/data/natural_earth2/*.shp`.
https://github.com/OSGeo/OSGeoLive/blob/master/app-data/qgis/QGIS-NaturalEarth-Example.qgz
 +- `/home/user/data/natural_earth2/ne_10m_lakes.shp`
 +- :

Reply:
Thank you for the feedback, we should change the path to read the shapefiles outside of /home

Reply:
@kalxas Oh, yes. Thanks for confirmation!

From local environment trial, changing `natural_earth2` folder's owner (`sudo chown -R root:users /usr/local/share/data/natural_earth2`) seems to be also necessary.

I am preparing the another PR about this (from the latest `master` branch), so please wait for it until tomorrow.